internal/app/router/delegating-router.go.go

```diff
@@ -0,0 +1,35 @@
+package router
```

Comment:
The name of the file has two .go suffixes.

Reply:
Thanks! :facepalm: